share/instruments/update_instruments_xml.py

```diff
@@ -159,12 +160,20 @@ def to_list(str):
     for instrument in instruments[group['id']].values():
         el = ET.SubElement(g_el, 'Instrument')
         to_attribute(el, instrument, 'id')
+        to_subelement(el, instrument, 'init') # must be first subelement
         to_subelement(el, instrument, 'family')
         to_comment(el, instrument, 'comment')
         to_subelement(el, instrument, 'trackName')
         to_subelement(el, instrument, 'longName')
         to_subelement(el, instrument, 'shortName')
-        to_subelement(el, instrument, 'init')
+        # if instrument["ddName"] != '[hide]':
+        #     to_subelement(el, instrument, 'trackName')
+        #     to_subelement(el, instrument, 'longName')
+        #     to_subelement(el, instrument, 'shortName')
+        #     if instrument["ddName"]:
+        #         dd_el = ET.SubElement(el, 'dropdownName')
+        #         dd_el.text = instrument["ddName"]
+        #         to_attribute(dd_el, instrument, 'ddMeaning', 'meaning')
```

Comment:
@RomanPudashkin, once this is merged, you can uncomment these 8 lines and delete the 3 lines immediately above (that set `trackName`, `longName` and `shortName` before the `if` statement), then rerun the script to generate the new style of instruments.xml that you will need to fix the dropdown in the New Score Wizard. It will fetch some new data from the online spreadsheet as well as adding the `<dropdownName>` element.